### DISCUSSION TITLE
Definitions for the luasocket library

### DISCUSSION THREAD

**Discussion by Josef-Friedrich** [LuaCATS/.github#2] [Transfers]
I created a repository containing the definitions for [luasocket](https://github.com/Josef-Friedrich/luasocket) (https://github.com/Josef-Friedrich/luasocket). I want to transfer this repo to LuaCATS.

**Comment by carsakiller** [comment]
Hello! Same process as [before](https://github.com/orgs/LuaCATS/discussions/1) 🙂

**Reply by Josef-Friedrich** [reply]
Yes
